test/recipes/15-test_genrsa.t

```diff
@@ -105,3 +115,25 @@ ok(!run(app([ 'openssl', 'genpkey', '-algorithm', 'RSA',
     ok(run(app([ 'openssl', 'rsa', '-check', '-in', 'genrsatest.pem', '-noout' ])),
        "rsa -check");
 }
+
+unless ($no_fips) {
+    my $provconf = srctop_file("test", "fips.cnf");
+    my $provpath = bldtop_dir("providers");
+    my @prov = ( "-use_libctx", "-provider_path", $provpath, "-config", $provconf );
+    my $infile = bldtop_file('providers', platform->dso('fips'));
+
+    ok(run(app(['openssl', 'fipsinstall',
+                '-out', bldtop_file('providers', 'fipsmodule.cnf'),
+                '-module', $infile,
+                '-provider_name', 'fips', '-mac_name', 'HMAC',
+                '-macopt', 'digest:SHA256', '-macopt', 'hexkey:00',
+                '-section_name', 'fips_sect'])),
+       "fipsinstall");
+
+    ok(run(app(['openssl', 'genpkey',
+                @prov,
+               '-algorithm', 'RSA',
+               '-pkeyopt', 'bits:2048',
+               '-out', 'genrsatest2080.pem'])),
```

Comment:
Shouldn't this say `bits:2080`?

Reply:
You are correct - I know I tested it locally :)